.github/ISSUE_TEMPLATE.md

```diff
@@ -4,15 +4,28 @@ Please don't file GitHub issues to ask questions! Instead use:
 * Stack Overflow: http://stackoverflow.com/questions/tagged/react-virtualized
 * The react-virtualized Slack channel: https://react-virtualized.now.sh
 
-# Reporting a Bug?
+**Do you want to request a *feature* or report a *bug*?**
 
-Please include either a failing unit test or a simple repro. You can start by forking one of these:
-* Code Sandbox: https://codesandbox.io/s/03qpzq1p9p?module=%2FExample.js
-* Plnkr: https://plnkr.co/edit/6syKo8cx3RfoO96hXFT1
-
-# Requesting a Feature?
+<!--
```

Comment:
Shall we just delete instead of comment?

Reply:
I'm not sure I understand the use of the comment <!-- ... --> tag

Even though these are markdown files, when you open a new issue- they're going to appear in plaintext. (See [here](https://github.com/bvaughn/react-virtualized/issues/new) for an example)

So the comment tag won't actually work, nor will any of the fancier formatting :)

Reply:
@bvaughn that's intentional. i think those are things we want people to think about, but text that we don't want showing up in the final issue that's opened, so if it's there as a comment we don't have to depend on people to delete the text. should keep the same intent of the original content without cluttering up issue descriptions.

Reply:
I see. I would expect people to delete the contents of the issue template (and replace with their own issue description, etc.)

It doesn't make much sense to leave the template content in the issue, I think.

Reply:
ok! happy to uncomment and just leave it there as guiding text